recipe_scrapers/quitoque.py

```diff
@@ -0,0 +1,90 @@
+from ._abstract import AbstractScraper
+from ._utils import csv_to_tags, get_minutes, get_yields, normalize_string
+
+
+class QuiToque(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "quitoque.fr"
+
+    @staticmethod
+    def _get_text(element):
+        if element:
+            return normalize_string(element.get_text())
+        else:
+            return None
+
+    def _get_time(self, time_name):
+        times = self.soup.select("div,.recipe-infos-short .item-info")
+        total_time = None
+        for time in times:
+            if time_name in time.get_text():
+                total_time = self._get_text(time).replace(time_name, "")
+        return get_minutes(total_time)
+
+    def _get_nutrient(self, nutrient_name):
+        nutrient_element = self._nutrients.find("p", string=nutrient_name).parent
+        return self._get_text(nutrient_element.find("p", class_="regular"))
+
+    def canonical_url(self):
+        return self.soup.find("meta", {"property": "og:url"}).get("content")
+
+    def author(self):
+        return "QuiToque"
+
+    def title(self):
+        return self._get_text(self.soup.find("h1", class_="title-2"))
+
+    def keywords(self):
+        product_tags = self.soup.find(id="product-tags").find_all(class_="badge")
+        keywords = ",".join(self._get_text(tag) for tag in product_tags)
```

Comment:
I'm not 100% sure about this -- your thoughts are welcome -- but I think it might be safer to filter out `None` values here and in similar string-join cases.

I would suggest:

```suggestion
        keywords = ",".join(normalize_string(tag.text) for tag in product_tags if tag and tag.text)
```

Reply:
yeah I aggree with @jayaddison  `if tag is None` probably would have been better but on the other hand if these things worry us _that_ much we are better of adjusting our linters check. I'll go ahead and merge, you've addressed plenty of other stuff already! Thanks